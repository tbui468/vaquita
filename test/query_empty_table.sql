create database sol;
open sol;

create table planets (
                        id int key,
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

select * from planets;

close sol;
drop database sol;
exit;
