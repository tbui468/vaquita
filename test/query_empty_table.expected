created database sol
opened database sol
created table planets
id, name, mass, number_of_moons, surface_pressure, ring_system
closed database sol
dropped database sol
